packages/dx-react-chart/docs/guides/legend_title.md

```diff
@@ -0,0 +1,27 @@
+# React Chart - Legend and Title
+
+The legend and title provide additional information about a chart.
+
+## Legend
+
+The Legend feature displays series on the current chart. Legend contains items. Items contain marker and label. Marker display series color. Label text equal series name.
+
+## Title 
+
+Title display helpful text. You must provide text to the title.
+
+## Related Plugins
+
+- `Legend` - renders a legend
+- `Title` - renders a title
+
+## Basic Setup
+Import the plugins listed above to set up a Chart with the legend and title.
+
+.embedded-demo({ "path": "chart-basic/basic-legend.wb3", "showThemeSelector": true })
+
+## Use custom legend markers
```

Comment:
Can we name this section 'Appearance Customization' and show how to customize both the `Title` and `Legend` plugins?